Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
new file mode 100644
--- /dev/null	(date 1760000964047)
+++ b/.gitignore	(date 1760000964047)
@@ -0,0 +1,15 @@
+# Gradle
+.gradle/
+build/
+*/build/
+
+# Kotlin
+.kotlin/
+
+# Kotlin JS store and related
+kotlin-js-store/
+kotlin-js/
+store/
+
+# Local machine-specific
+local.properties
